apps/angular/src/app/features/anime-table/anime-table.component.html

```diff
@@ -0,0 +1,25 @@
+<table mat-table [dataSource]="(anime$ | async) ?? []" class="anime-table">
+	@for (column of displayedColumns; track $index) {
+		<ng-container [matColumnDef]="column.key">
+			<th mat-header-cell *matHeaderCellDef>{{column.header}}</th>
+			<td mat-cell *matCellDef="let element">
+				@switch (column.key) {
+					@case (columnKey.Image) {
+						<img [src]="element.image" class="table-image" [alt]="element.titleJpn"/>
+					}
+					@case (columnKey.AiredStart) {
+						{{element.aired.start | date:"dd.MM.yyyy" | empty}}
```

Comment:
I think you will display date in other components as well. Usually all dates are display in the same format for consistency.
In our projects we often create a constant variable for date format and use it everywhere. It will look like this:
```suggestion
						{{element.aired.start | date:dateFormat | empty}}
```